Added definition of Holder (#306)
italia/eidas-it-wallet-docs: docs/en/defined-terms.rst

     - The degree of confidence in the vetting process used to establish the identity of the User and the degree of confidence that the User who presents the credential is the same User to whom the Digital Credential was issued.
   * - Holder Key Binding
     - Ability of the Holder to prove legitimate possession of the private part, related to the public part attested by a Trusted Third Party.
   * - Holder

>      - User that has activated and has full control of an instance of the EUDI wallet and the related Private Keys.

Reviewer: ```suggestion
     - Natural or Legal person that receives Verifiable Credentials and has control over them to present them to the Verifiers as Presentations, using the Wallet. The Holder is the User in control of the Wallet.
```

>      - Natural or Legal person that receives Verifiable Credentials and has control over them to present them to the Verifiers as Presentations, using the Wallet. The Holder is the User in control of the Wallet.

Reviewer: ```suggestion
     - Natural or Legal person that receives Verifiable Credentials from the Credential Issuers, manages the Verifiable Credentials within the Wallet, and presents them to Verifiers. The Holder is the User in control of the Wallet.
```
